tests/topotests/bgp_always_compare_med/test_bgp_always_compare_med_topo1.py

```diff
@@ -0,0 +1,1122 @@
+#!/usr/bin/env python3
+#
+# SPDX-License-Identifier: ISC
+#
+# Modified work Copyright (c) 2019 by VMware, Inc. ("VMware")
+# Original work Copyright (c) 2018 by Network Device Education
```

Comment:
What part of _original_ work is done? Can we drop this statement at all?

Reply:
Updated it.